### PR TITLE
RCD upgrades have names and icon states

### PR DESCRIPTION
**Pull request by lessthnthree**
## About The Pull Request

Adds names and icon states to RCD/RPD upgrades

## Why It's Good For The Game

I don't have a pile of 6 disks on the lathe that all look identical and the RCD going YOU HAVE THAT ONE INSTALLED ALREADY, TRY AGAIN IDIOT as I fall into further despair

## Changelog

:cl: LT3
qol: The various RCD upgrade disks no longer look identical
/:cl:

### CODE REVIEW

**Comment by newfren** [comment]
would be nice if you could tell which disks had already been installed in an rcd

**Comment by lessthnthree** [comment]
> would be nice if you could tell which disks had already been installed in an rcd

Yeah I was planning to do that in a separate PR.

File: code/game/objects/items/rcd/RHD.dm [hunk 1]
```diff
@@ -290,27 +290,39 @@
...
 	upgrade = RCD_UPGRADE_FRAMES
 
 /obj/item/rcd_upgrade/simple_circuits
+	name = "RCD advanced upgrade: simple circuits"
 	desc = "It contains the design for firelock, air alarm, fire alarm, apc circuits and crap power cells."
+	icon_state = "datadisk4"
 	upgrade = RCD_UPGRADE_SIMPLE_CIRCUITS
 
 /obj/item/rcd_upgrade/anti_interrupt
+	name = "RCD advanced upgrade: anti disruption"
```
Comment (SyncIt21) [inline code/game/objects/items/rcd/RHD.dm:305]: Something like "RCD Anti Disruption Upgrade" and similar names for other's sound more meaningful.

Comment (lessthnthree) [reply code/game/objects/items/rcd/RHD.dm:305]: Items don't use capital case unless they're proper nouns.

Comment (SyncIt21) [reply code/game/objects/items/rcd/RHD.dm:305]: I wasn't talking about the case, just the wording

**Pull request merged**